docs/labs/15-Figma.md

```diff
@@ -0,0 +1,330 @@
+# Figma Lab
+
+- [Overview](#overview)
+- [Creating Your Figma Account](#creating-your-figma-account)
+- [Creating Your First Figma Project](#creating-your-first-figma-project)
+- [Adding Elements to Project](#adding-elements-to-project)
+    - [Adding a Rectangle](#adding-a-rectangle)
+    - [Adding a Circle](#adding-a-circle)
+    - [Adding a Label](#adding-a-label)
+- [Adjusting Properties](#adjusting-properties)
+    - [Changing Corners](#changing-corners)
+    - [Changing The Label](#changing-the-label)
+    - [Changing Font](#changing-font)
+    - [Changing Colors](#changing-shape-colors)
+- [Challenge Object 1](#challenge-object-1)
+- [Challenge Object 2](#challenge-object-2)
+- [Porting Objects to a ACM Java Project](#porting-objects-to-a-acm-java-project)
+    - [Exporting as attributes](#exporting-as-attributes)
+    - [Exporting as an Image](#exporting-as-an-image)
+- [Drawbacks](#drawbacks)
+    - [Gradients](#gradients)
+    - [Mixed Colors on Labels](#mixed-colors-on-labels)
+    - [And More..](#and-more)
+- [What Next..](#what-next)
+
+## Overview
+
+Have you heard of Figma before this class? If not, this lab is for you.
+
+This lab will be teaching you the simplest core functions of figma to
+help you create a seamless, connected UI across all screens in your Comp 55 Projects.
+
+Figma is a powerful tool to help you line up your ideas before you
+begin coding them, which is important if you generally like
+to jump in and begin programming.
+
+Upon completing this lab you will know how to:
+
+- Create a figma project
+- Add objects onto your figma project
+- Edit object's properties
+- Be able to port the design into a java project
+
+Submission for this lab will be:
+// TODO am I even allowed to add submissions in the first place?
+
+## Creating Your Figma Account
+
+To Get Started with this lab, we will assume you've never touched figma
+before and need to start from the first opening of the website.
+
+Navigate to [Figma](https://www.figma.com/) and then continue.
+
+You'll immediately see a pretty website however we want to just create an account.
+
+Find the button in the upper right corner that says "Get started for free"
+
+![figma lab free account image](lab15media/figma-lab-get-started-for-free.png)
+
+I would reccomend just using your school email, so just click on "Continue with Google"
+
+### Google Button
+
+Continuing from before..
+
+Click on your signed in school account.
+
+Then it should take you into your home screen.
+
+Now you have successfully created your Figma account.
```

Comment:
I wouldn't leave the spaces in here in between all of these, or I would make these an ordered or unordered list, isntead of having single blank lines throughout.

Because these feel like steps let's use one of those list markers.